index.js

```diff
@@ -0,0 +1,61 @@
+/* globals Vue */
+
+new Vue({
+  el: '#app',
+  data: {
+    additionalText: '',
```

Comment:
新規登録ときだけに使う値ならこういう名前の方がシンプルかも？

```suggestion
    newText: '',
```